.github/workflows/verify-gdrive.yml

```diff
@@ -4,15 +4,21 @@ on:
 jobs:
   check:
     runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/caciviclab/disclosure-backend-static/${{github.ref_name}}:latest
```

Comment:
What happens here if no image has been pushed for `github.ref_name`?

Reply:
It should fail.  Good question, however.  I've updated main.yml to build the image as soon as a new branch is created.  Since the above is a manually run workflow, you would have to be quick to run it if you wanted to generate the failure.